src/components/CoinsList.js

```diff
@@ -1,20 +1,35 @@
 import { useDispatch, useSelector } from 'react-redux';
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
 import { fetchCoins } from '../redux/feature/coinSlice';
 import CoinsCard from './coinsCard';
 
 const CoinsList = () => {
-  const coins = useSelector((store) => store.coin.coins);
+  const { coins, isLoading } = useSelector((store) => store.coin);
+  const [filter, setFilter] = useState('');
+  // eslint-disable-next-line
```

Comment:
- [ ] Kindly remove disable linters as they help us to write code in the standard accepted way